docs/docs/code/python/import-mappings/README.md

```diff
@@ -0,0 +1,10 @@
+---
+content_class: max-w-max
```

Comment:
Was there a reason you set this? When I navigated to this page I noticed the difference. If you end up removing the list of mappings, do you still need the page this wide?

Reply:
I fixed the width to be slightly larger than the max width of other pages, give it another shot. There was just an issue with the importing table collapsing on larger viewports.